README.md

```diff
@@ -16,17 +16,7 @@ linking for LoopBack applications.
 
 which will allow the user to link the social media accounts that they want as well as the users could sign up with the same email account that is used for facebook/twitter/google/local if they wish to keep them separate.
 
-If more info is required from the Facebook profile such as email, it could still be obtained. In `node_modules\passport-facebook\lib\strategy.js`, replace:
-
-```
-this._profileURL = options.profileURL || 'https://graph.facebook.com/me';
-```
-
-with
-
-```
-this._profileURL = options.profileURL ||
-    'https://graph.facebook.com/v2.2/me?fields=first_name,gender,last_name,link,locale,name,timezone,verified,email,updated_time';
-```
+If more info is needed from the Facebook profile(email, gender, timezone, etc), it could still be obtained, see
```

Comment:
Maybe just change this whole sentence instead:

Facebook profile information (such as email, gender, timezone, etc) may still be included if necessary. See ...